context.copytostorage/addon.py

```diff
@@ -0,0 +1,57 @@
+import xbmc
+import xbmcgui
+import xbmcaddon
+import sys
+import os
+import errno
+import shutil
+
+__addon__ = xbmcaddon.Addon()
+__addonname__ = __addon__.getAddonInfo('name')
+
+def copyFileOrFolder(src, dest):
+    print("(src, dest)",src, dest)
+    xbmc.executebuiltin("Notification(\"Started Copying ...\", \"%s\")" % sys.listitem.getLabel())
+
+    try:
+        shutil.copytree(src, dest)
```

Comment:
usually it is better to use `xbmcvfs` since the src or dst can be a virtual path (e.g. nfs://)